cli/cmd/inject.go

```diff
@@ -36,6 +36,7 @@ const (
 	// for inject reports
 	hostNetworkDesc = "hostNetwork: pods do not use host networking"
 	unsupportedDesc = "supported: at least one resource injected"
+	udpDesc         = "udp: pods do not use UDP protocol"
```

Comment:
I am wondering if this should be a statement like `all pods using TCP protocol`.

Reply:
yeah, i went back and forth on this one. i'm a bit hesitant to use a phrase like `all pods`, because whether a pod has zero, one, or all pods set as UDP, we're still going to inject. the intent is more just a warning to the user that _at least one_ pod is using UDP.